### DISCUSSION TITLE
ECHIPA LABORATORUL DEXTER

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#114] [TEAMs]
David Mondoc
role: Scraper Developer with Python
discord: david mondoc

**Comment by sebiboga** [comment]
Dragoi Diana Lavinia
role: Manual Tester
discord: deedee_diana

**Comment by sebiboga** [comment]
? automation tester???

**Reply by davidmondoc** [reply]
nu avem
